Gemfile

```diff
@@ -79,3 +80,6 @@ end
 gem 'net-imap'
 gem 'net-pop'
 gem 'net-smtp'
+
+# Use Redis for Action Cable
```

Comment:
Surprised we didn't have this already, where were Sidekiq workers running before?

Reply:
Good question mate, Sidekiq doesn't need the Redis gem for most use cases https://github.com/mperham/sidekiq/wiki/Using-Redis. 

It connects directly to the default Redis port in development (localhost:6379). In production it looks for a `REDIS_URL` env var we have set up.